index.html

```diff
@@ -0,0 +1,218 @@
+             both academy and industry by covering not only basic
+             and applied research,
+             but also philosophical and ethical issues
+             regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/exhibition-icon.png"
+                alt="exhibition"
+                class="icon"
+              />
+              <p class="program-name">CC Exhibition</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry 
+             by covering not only basic and applied research,
+             but also philosophical and ethical issues
+              regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/forum-icon.png" alt="forum" class="icon" />
+              <p class="program-name">Forum</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical 
+             issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img
+                src="./assets/images/workshop-icon.png"
+                alt="workshop"
+                class="icon"
+              />
+              <p class="program-name">Workshop</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+             both academy and industry by covering
+              not only basic and applied research,
+             but also philosophical and ethical
+              issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+          <li class="item">
+            <div class="image">
+              <img src="./assets/images/ignite-icon.png" alt="ignite" class="icon" />
+              <p class="program-name">CC Ignite</p>
+            </div>
+            <p class="program-description">
+              This second edition aims at joining
+              both academy and industry by covering
+               not only basic and applied research,
+              but also philosophical and ethical
+               issues regarding the future of humanity in an AI world.
+            </p>
+          </li>
+        </ul>
+        <div class="whole-program"><a href="#">SEE THE WHOLE PROGRAM</a></div>
+        <button type="button" class="join">Join Artificial Intelligence
+          International Conference</button>
+      </section>
+
+      <section class="featured-speakers">
```

Comment:
`Featured Speakers` should be the same as in the design. So you can implement it using Grid. There are two columns and three rows in the design. You may have two rows and two columns. So kindly see [this](https://www.w3schools.com/css/css_grid.asp) resource and implement it.